community/emacs/depends

```diff
@@ -1,2 +1,4 @@
-ncurses
 zlib
+ncurses
+libxaw3d
+imagemagick6
```

Comment:
Does the dependency list need updating too?

Reply:
uhhhhhhh maybe